lib/src/test/java/vmath/Mat3Test.java

```diff
@@ -7,6 +7,31 @@ public class Mat3Test {
 
     @Test
     void givenNoArgs_whenMat3Created_thenMatrixIsIdentity() {
+        var m1 = new Mat3();
+
+        assertEquals(1.0f, m1.get(0, 0));
+        assertEquals(0.0f, m1.get(0, 1));
+        assertEquals(0.0f, m1.get(0, 2));
+        assertEquals(0.0f, m1.get(1, 0));
+        assertEquals(1.0f, m1.get(1, 1));
+        assertEquals(0.0f, m1.get(1, 2));
+        assertEquals(0.0f, m1.get(2, 0));
+        assertEquals(0.0f, m1.get(2, 1));
+        assertEquals(1.0f, m1.get(2, 2));
+    }
+
+    @Test
+    void givenIndices_whenGetCalled_thenReturnCorrectValue() {
         //TODO
     }
+
+    @Test
+    void givenInvalidIndices_whenGetCalled_thenThrowIllegalArgumentException() {
+        var m1 = new Mat3();
+
+        assertThrows(IndexOutOfBoundsException.class, () -> m1.get(-1, 0));
```

Comment:
Please fix the name of this test: it checks for whether get() throws IndexOutOfBoundsException, not IllegalArgumentException.

Reply:
done